RSN for removing cugraph-ops (#566)
rapidsai/docs: _notices/rsn0041.md

---
layout: notice
parent: RAPIDS Support Notices
grand_parent: RAPIDS Notices
nav_exclude: true
notice_type: rsn
# Update meta-data for notice
notice_id: 41 # should match notice number
notice_pin: true # set to true to pin to notice page

> title: "Remove cugraph-ops from RAPIDS"

Reviewer: ```suggestion
title: "Removing cugraph-ops from RAPIDS"
```

I think "removing" is a bit more consistent with the way the other things at https://docs.rapids.ai/notices/ are written.

notice_author: RAPIDS Ops
notice_status: Completed
notice_status_color: green
# 'notice_status' and 'notice_status_color' combinations:
#   "Proposal" - "blue"
#   "Completed" - "green"
#   "Review" - "purple"
#   "In Progress" - "yellow"
#   "Closed" - "red"
notice_topic: Platform Support Change
notice_rapids_version: "v25.02+"
notice_created: 2025-01-10
# 'notice_updated' should match 'notice_created' until an update is made
notice_updated: 2025-01-10
---

## Overview

> RAPIDS will stop releasing cugraph-ops in 25.02, including the `libcugraphops` and `pylibcugraphops` package. The graph sampling code previously used by cugraph has been directly integrated into cugraph. The equivariant neural network primitives have been migrated to BioNeMo, with the front-end package now available as [cuEquivariance](https://github.com/NVIDIA/cuEquivariance). The GNN aggregation primitives are also set to be relocated to a new destination soon.

Reviewer 2: ```suggestion
RAPIDS will stop releasing `cugraph-ops` in 25.02, including the `libcugraphops` and `pylibcugraphops` packages. The graph sampling code previously used by cuGraph has been directly integrated into cuGraph. The equivariant neural network primitives have been migrated to BioNeMo, with the front-end package now available as [cuEquivariance](https://github.com/NVIDIA/cuEquivariance). The GNN aggregation primitives will be relocated to a new destination soon.
```

## Impact

> The GNN convolutional layers in the `cugraph_dgl.nn` and `cugraph_pyg.nn` modules will be temporarily removed. They will be reintroduced once the migration is complete.

Reviewer 2: It's a little uncertain what the migration will look like yet, so let's just say "at a later time."
```suggestion
The GNN convolutional layers in the `cugraph_dgl.nn` and `cugraph_pyg.nn` modules will be temporarily removed. They will be reintroduced at a later time.
```
